### ISSUE TITLE
Feature request: Create @accessibility Slack team

### ISSUE THREAD

**Issue by lukefretwell** [CivicActions/accessibility#367]
**For whom is this feature?**
A clear and concise user story.
As CivicActioner, I want to be able to easily ask an accessibility question in the context of a Slack conversation, so that I so that I can get support quickly and seamlessly.

**Describe the solution you'd like**

- [ ] Create an @accessibility Slack team and communicate this to the company
- [ ] Add to a /help page (+ include link to GitHub discussions, email, contact form)
